man/systemd.netdev.xml

```diff
@@ -504,6 +504,15 @@
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><varname>DiffServiceCodePoint=</varname></term>
+        <listitem>
+          <para>A boolean. When true, inherits Differentiated
+          Service Code Point (DSCP) field between inner and outer
+          header for ip6_tunnel.</para>
```

Comment:
inherits *the* ...

Also, could you give a little more background here on what the effect of this flag is?